src/material/list/list.scss

```diff
@@ -17,6 +17,14 @@ a.mdc-list-item--activated {
 // explicitly set `display: block`
 .mat-mdc-list-base {
   display: block;
+
+  // MDC sets `pointer-events: none` on these elements,
+  // even though we allowed interactive content in them.
+  .mdc-list-item__start,
+  .mdc-list-item__end,
+  .mdc-list-item__content {
```

Comment:
Just double checking: Should content allow for pointer-events:none? I don't fully recall without looking, but might be needed for anchor elements and disabling?

Reply:
MDC's code doesn't have any comments on why the `pointer-events: none` is there. I don't think that it's there for anchor list items that are disabled, because it's only a wrapper around the projected content. My best guess is that it's used as a shorthand to switch the cursor to the default and disable text selection.

Reply:
👍